test/config/integration_tests.yml

```diff
@@ -142,17 +142,6 @@ testCases:
     kwargs:
       import_data_file_path: config/nnictl_experiment/test_import.json
 
-- name: nnicli
```

Comment:
similar test should be added back to test the refactored code?

Reply:
we don't support launch from config file and stop a connected experiment right now, will add back in next pr.